src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Utilities/EnvironmentHelper.cs

```diff
@@ -15,6 +15,14 @@ internal class EnvironmentHelper : IEnvironmentHelper
 
         public string ExpandEnvironmentVariables(string name)
         {
+            if (name.IndexOf('%') == -1)
```

Comment:
nit: You can just use `Contains` since it does the same check without exposing the mechanics of `IndexOf`.

https://github.com/microsoft/referencesource/blob/5697c29004a34d80acdaf5742d7e699022c64ecd/mscorlib/system/string.cs#L2172-L2174

Reply:
I'd prefer `Contains` too, but in .NET Framework it a string, not a character. Searching for a string is more involved than searching for a character. This implementation is likely to be snappier.